config/initializers/000-zeitwerk.rb

```diff
@@ -54,3 +54,12 @@ def self.inflect(overrides)
   "lib/i18n/backend",
   "lib/unicorn_logstash_patch.rb",
 )
+
+# Always load problem checks. Problem checks are registered when their classes are
+# loaded, which means if we're not eager loading, the registry would be empty.
+#
+if !Rails.application.config.eager_load
+  Rails.application.config.to_prepare do
+    Rails.autoloaders.main.eager_load_dir("#{Rails.root}/app/services/problem_check")
+  end
+end
```

Comment:
Do we consider adding `app/services` to the autoload path instead?

Reply:
That doesn't solve the problem. The problem check is registered when the class is loaded. If the file is never loaded, then the class is never referenced, and so is never put in the registry.